src/VPolygon.jl

```diff
@@ -553,3 +553,52 @@ function translate(P::VPolygon{N}, v::AbstractVector{N}) where {N<:Real}
                                 "set by a $(length(v))-dimensional vector"
     return VPolygon([x + v for x in vertices_list(P)])
 end
+"""
+    minkowski_sum(P::VPolygon{N}, Q::VPolygon{N}) where {N<:Real}
+
+The Minkowski Sum of two polygon in vertex representation.
+
+### Input
+
+- `P` -- polygon in vertex representation
+- `Q` -- another polygon in vertex representation
+
+### Output
+
+A polygon in vertex representation.
+
+### Algorithm
+
+We treat each edge of the polygons as a vector, attaching them in polar order
+(attaching the tail of the next vector to the head of the previous vector). The
+resulting polygonal chain will be a polygon, which is the Minkowski sum of the given polygons.
+"""
+
+function minkowski_sum(P::VPolygon{N}, Q::VPolygon{N}) where {N<:Real}
+    vlP = vertices_list(P)
+    vlQ = vertices_list(Q)
+    mP = length(vlP);
+    mQ = length(vlQ);
+    i = 1
+    k = 1
+    j = 1
+    R = Vector{Vector{Float64}}(undef, mP+mQ)
+    fill!(R, N[0, 0])
+    while i <= size(R, 1)
+        OP = vlP[(k%mP+1)]-vlP[(k-1)%mP+1]
+        OQ = vlQ[(j%mQ+1)]-vlQ[(j-1)%mQ+1]
+        R[i] = vlP[(k-1) % mP+1] + vlQ[(j-1) % mP+1]
+        is_rightTurn = LazySets.right_turn(OP, OQ, N[0, 0])
```

Comment:
```suggestion
        is_rightTurn = right_turn(OP, OQ, N[0, 0])
```

(since this function is inside the `LazySet` library, you don't need to write the module's name -- it is needed for external use of `right_turn` since this function is not exported; see [this part of the julia docs for examples](https://docs.julialang.org/en/v1/manual/modules/index.html#modules-1))

Reply:
The name `is_rightTurn` suggests that the type of that variable is `Bool`. But `right_turn` returns a number. I admit that the documentation of this function is wrong (I created #1379 to fix that). To help intuition, I would use a different name (e.g., just `turn`) and instead add a comment to each case that explains what it corresponds to.